es_query_executor/queries/job_status-current/jobs_nominal_old.json

```diff
@@ -0,0 +1,41 @@
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "bool": {
+            "should": [
+              {
+                "match": {
+                  "status": "job-completed"
+                }
+              },
+              {
+                "match": {
+                  "status": "job-revoked"
+                }
+              },
+              {
+                "match": {
+                  "status": "job-deduped"
+                }
+              },
+              {
+                "match": {
+                  "status": "job-failed"
```

Comment:
We don't want job-failed within this nominal deletion policy. We already have jobs_failed_old.json for the failed jobs.